### ISSUE TITLE
iD discards information when merging point and way

### ISSUE THREAD

**Issue by boothym** [openstreetmap/iD#7446]
I saw a node with the name of a building also tagged with amenity=* but no building drawn on the map.

So I drew the building outline shape, shift clicked to select both and then pressed C to merge. iD merged the two, adding a building=yes tag as well.

However the amenity tag value was simply replaced by amenity=yes. It seems that this only happens if the amenity value is not a preset, as it did not occur if I added an amenity=cafe node and then merged with a way.

**Basically, iD should not discard tag values when merging.**

**Comment by quincylvania** [comment]
Thanks a lot for reporting, @boothym, we take potential data loss seriously.
